libs/ui-watt/src/lib/foundations/typography/index.scss

```diff
@@ -0,0 +1,92 @@
+body h1,
```

Comment:
These styles should also be available in either css-vars or some scss function so they can be received inside of a stylesheet of a component

Reply:
Question: Do you think that would be necessary? Since these are global styles, they will just "apply" everywhere. Unless of course there's another style that overwrites them. If the Angular theme is configured to use the same font, things will "just work". And besides the tag (`h1`) selector , there's also a `.watt-heading-1` class in case someone wants a `p` that looks like a heading (which we must be careful of :D)

Reply:
That's true, lets skip it until it's needed